shoes-core/lib/shoes/common/clickable.rb

```diff
@@ -5,10 +5,12 @@ module Clickable
 
       def click(&blk)
         @gui.click blk
+        self
       end
 
       def release(&blk)
         @gui.release blk
+        self
```

Comment:
I love returning self to allow chainability, we might even want to do it in more/other places :) But, some other time you know!